src/style.css

```diff
@@ -0,0 +1,112 @@
+body,
+div,
+p,
+a {
+  margin: 0;
+  padding: 0;
+  color: #060b35;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}

```

Comment:
set `border-box` to the * selector, font-family to html, and inherit where needed. Not all of these elements have default margins and paddings. try to improve this section of code to make better performance